metadata-ingestion/setup.py

```diff
@@ -321,6 +321,7 @@ def get_long_description():
             "hive",
             "starburst-trino-usage",
             "powerbi",
+            "great-expectations",
```

Comment:
why is great-expectations plugin added as a default requirement? Since it is not required to be installed for pull-based ingestion, I think we should keep it out.